From 741505f1e643d4bf887060d70607e5e763127e61 Mon Sep 17 00:00:00 2001
From: Jonas Witschel <diabonas@archlinux.org>
Date: Sat, 2 Nov 2019 17:56:48 +0100
Subject: [PATCH] Use system gtest instead of building from submodule

This requires gtest version >=1.10.0 since lib3mf uses the new name
INSTANTIATE_TEST_SUITE_P instead of the deprecated
INSTANTIATE_TEST_CASE_P.
---
 Tests/CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/Tests/CMakeLists.txt b/Tests/CMakeLists.txt
index de62d3a..0ae693b 100644
--- a/Tests/CMakeLists.txt
+++ b/Tests/CMakeLists.txt
@@ -1,10 +1,5 @@
-ADD_SUBDIRECTORY (googletest EXCLUDE_FROM_ALL)
 enable_testing()
 
-SET_TARGET_PROPERTIES (gtest PROPERTIES FOLDER gtest)
-SET_TARGET_PROPERTIES (gtest_main PROPERTIES FOLDER gtest)
-
-SET(gtest_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/googletest/googletest")
 add_definitions( -DTESTFILESPATH="${CMAKE_CURRENT_SOURCE_DIR}/TestFiles/")
 add_definitions( -DLTESTFILESPATH=L"${CMAKE_CURRENT_SOURCE_DIR}/TestFiles/")
 add_definitions( -DLOUTFILESPATH=L"${CMAKE_BINARY_DIR}/")
-- 
2.23.0

